scripts/util.js

```diff
@@ -1,11 +1,18 @@
+const Settings = require('./settings.js');
+const readline = require('readline');
+
 var Util = function(){};
 
 Util.refreshConsole = function(){
-  // var lines = process.stdout.getWindowSize()[1];
-  // for(var i = 0; i < lines; i++) {
-  //     console.log('\r\n');
-  // }
   process.stdout.write('\033c');
 };
 
+Util.overwriteConsole = function(){
+  if (Settings.getInstance().properties['disableColors']) {
+    process.stdout.write('\033c');
+  } else {
+    readline.cursorTo(process.stdout, 0, 0);
+  }
+};
+
```

Comment:
The people I know who use this with colours disabled do so because their terminals didn't support the colour escape codes, so I thought it'd make sense to put this in there too. Think it should be its own setting?